Components/FPSCounter.cs

```diff
@@ -0,0 +1,82 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public static class FPSCounter
+{
+    public static int s_smoothingWindowSize = 100;
+
+    private static Queue<float> s_measures = new Queue<float>(s_smoothingWindowSize);
+    private static int s_lastProcessedFrame = -1;
+    private static int s_lastCalculatedFrame = -1;
```

Comment:
`s_lastCalculatedFrame` => `s_lastProcessedFrame`, since it marks the frame that had its data processed